sdk/compute/arm-compute/test/compute_examples.ts

```diff
@@ -261,7 +261,7 @@ describe("Compute test", () => {
         ],
       }
     }, testPollingOptions)
-    assert.equal(res.type, "Microsoft.Compute/virtualMachines");
+    assert.equal(res.type, undefined);
```

Comment:
I think it is wrong, because in recording it is "Microsoft.Compute/virtualMachines". But in test,it should be undefined